### PR TITLE
Rya 398

### PR DESCRIPTION
**Pull request by isper3at**
## Description
Changed beforeclass to before, removed System.out.println()'s

### Tests
N/A

### Links
[Jira](https://issues.apache.org/jira/browse/RYA-398)

### Checklist
- [ ] Code Review
- [x] Squash Commits

#### People To Reivew
@jdasch 
@meiercaleb

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/516/

File: extras/indexing/src/test/java/org/apache/rya/indexing/mongo/MongoEntityIndexTest.java [hunk 1]
```diff
@@ -54,11 +55,11 @@
                     .build());
     private static final RyaURI RYA_PERSON_TYPE = new RyaURI("urn:person");
 
-    private static EntityIndexOptimizer optimizer;
-    private static EntityStorage entityStorage;
+    private EntityIndexOptimizer optimizer;
+    private EntityStorage entityStorage;
 
-    @BeforeClass
-    public static void beforeClass() throws Exception {
```
Comment (meiercaleb) [inline extras/indexing/src/test/java/org/apache/rya/indexing/mongo/MongoEntityIndexTest.java]: Decided to change this to @Before as opposed to changing parent init to @BeforeClass?

**Comment by jdasch** [comment]
RYA-397 was the rename of mockmongo stuff,  RYA-398 was fixing the `MongoEntityIndexTest`

**Comment by jdasch** [comment]
Caleb, RYA-353 handles the Test to IT refactor.

**Comment by jdasch** [comment]
Oh, and this built for me fine.

**Pull request closed without merging**